using/cmdline.po

```diff
@@ -839,11 +841,13 @@ msgid ""
 "``True``"
 msgstr ""
 "Définit l'attribut :attr:`~sys.flags.dev_mode` de :attr:`sys.flags` à "
-"``True``."
+"``True``"
 
 #: ../Doc/using/cmdline.rst:458
 msgid ":class:`io.IOBase` destructor logs ``close()`` exceptions."
 msgstr ""
+"Le destructeur de :class:`io.IOBase` journalise les exceptions levées par "
```

Comment:
https://imgflip.com/i/3cyxnm

Reply:
😆

Reply:
mdrrr

Reply:
j'ai déjà eu à traduire : "to log", et c'est pas évident. Moi j'aime bien "journalise" mais je pense que c'est incompréhensible pour les 3/4 des gens et que c'est pas vraiment du français correct. En général je traduis par "enregistre/écrit/consigne". Gardons-là ici, mais faudra fixer un terme un jour :)